packages/@aws-cdk/custom-resource-handlers/test/aws-s3/notifications-resource-handler/test_index.py

```diff
@@ -86,337 +83,330 @@ def merge_notification_configurations(conf1: Dict, conf2: Dict):
 
 
 class ManagedBucketTest(unittest.TestCase):
-    @patch("index.put_bucket_notification_configuration")
+    @patch('index.s3')
```

Comment:
There was a function that executes the `put_bucket_notification_configuration` method of `boto3`, but it has been removed.